lib/src/tweet_video.dart

```diff
@@ -44,6 +45,7 @@ class _TweetVideoState extends State<TweetVideo>
       },
       aspectRatio: widget.tweetVM.getDisplayTweet().videoAspectRatio!,
       controlsConfiguration: BetterPlayerControlsConfiguration(
+        controlBarColor: widget.controlBarColor??Colors.black.withOpacity(0.5),
```

Comment:
the code doesn't look formatted. We should have spaces inbetween

Reply:
Also as I see in the package the default color they use is `black87` I think we should use the same as default here,